src/modules/rlm_redis_ippool/rlm_redis_ippool_tool.c

```diff
@@ -1588,6 +1687,26 @@ do { \
 	}
 		continue;
 
+	case IPPOOL_TOOL_ASSIGN:
```

Comment:
I think we need an unassign function too to remove the staticness?  release kinda does something different.  You also need to update the stats, to differentiate between lease times with the static flag set.